### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by RuoyuDeng** [sdushantha/wifi-password#42]
Hi,

I do not know why no matter what SSID I put in as input, it keeps saying IndexError for some reason.
![image](https://user-images.githubusercontent.com/71870877/107325636-122e2080-6ae5-11eb-9228-1c2da1a7b73c.png)

my `OS` is `Windows 10`
my python version is `3.9.0`
my `wifi-password` version is `1.0.9`

**Comment by lixiaoyi01** [comment]
win10 要用gbk解析，并且获取得密码串需要改一下正则 re.findall(r"Key Content\s+:\s(.*)", password) =>re.findall(r"关键内容\s+:\s(.*)", password)

**Comment by RuoyuDeng** [comment]
请教一下，gbk解析是什么意思呢？是在cmd里执行的吗？

**Comment by lixiaoyi01** [comment]
1、
![image](https://user-images.githubusercontent.com/41557247/107338111-24fd2100-6af6-11eb-94fd-3cbbcbe4e620.png)
2、
![image](https://user-images.githubusercontent.com/41557247/107338183-36dec400-6af6-11eb-8a9d-a88ea59cefc6.png)
需要修改这两块、我没有权限修改。

**Comment by RuoyuDeng** [comment]
你好，我根据你的代码改了之后
1. 
![image](https://user-images.githubusercontent.com/71870877/107347578-ecaf1000-6b00-11eb-818b-7e2715528e7b.png)

2. 
![image](https://user-images.githubusercontent.com/71870877/107347651-005a7680-6b01-11eb-9d40-9800415809bd.png)
在原安装路径下执行`python setup.py install`重新依据新的wifi_password.py文件再次安装后，问题仍然存在：
![image](https://user-images.githubusercontent.com/71870877/107348333-d6ee1a80-6b01-11eb-99a8-77d63577d7f3.png)
请问可能是什么原因呢？

**Comment by sunrabbit123** [comment]
Enter the following syntax into the window shell.
```shell
netsh wlan show profiles name="SSID" key=clear
```
And please check what Key Contents has been translated into.

And replace it.

This is covered in #39.

**Comment by RuoyuDeng** [comment]
Hi, I think I am a bit lost when you said: "replace it". This might sound stupid but where should I replace what to what exactly?

**Comment by sunrabbit123** [comment]
this is example code

```python
elif platform == constants.WINDOWS:
                password = Utils.run_command(
                    f'netsh wlan show profile name="{ssid}" key=clear | findstr Key'
                )

                if password != "":
                    password = re.findall(r"Key Content\s+:\s(.*)", password)[0]
```

you can find it(Key Content, Key)

**Comment by RuoyuDeng** [comment]
I have figured out why it happened. Since my computer is in Chinese, so I have to change the "Key" in 
`f'netsh wlan show profile name="{ssid}" key=clear | findstr Key'` to Chinese, which is "关键内容“. Now it works pefectly.

**Comment by zxc120389574** [comment]
仅仅针对Windows10，我修改了下wifi_password.py文件。自己测试是成功的，有兴趣你可以用文件对比工具对比下修改的内容。
文件地址在这里：https://note.youdao.com/ynoteshare1/index.html?id=d130d9fdb160e98a44e7032cddd2881b&type=note

**Comment by sdushantha** [comment]
This is happening due to language issues. Thanks to @celianvdb, it has been fixed in 72b299d
